runtime/doc/nvim_python.txt

```diff
@@ -27,7 +31,15 @@ simple tutorial:
   >
       $ pip install neovim
 <
-Most Python plugins created for Vim 7.3 should work after these steps.
+						       *nvim-python3-quickstart*
 
+If you just want to start using Vim Python3 plugins with Nvim quickly, here's
+a simple tutorial:
+
+- Make sure Python 3.3 or above is available in your `$PATH`
+- Install the `neovim` Python3 package:
+  >
+      $ pip3 install neovim
+<
```

Comment:
Given how similar these instructions are, I think this could just be merged into the above section, so we only need `nvim-python-quickstart`. The whole section could look like something this:

```
2. Quickstart						 *nvim-python-quickstart*

To use Vim Python 2/3 plugins with Nvim, do the following:

- For Python 2 plugins, make sure an interpreter for Python 2.6 or 2.7 is available in your `$PATH`, then
install the `neovim` Python package:
  >
      $ pip install neovim
<

- For Python 3 plugins, make sure an interpreter for Python 3.3 or above is available in your `$PATH`, then
install the `neovim` Python package:
  >
      $ pip3 install neovim
<
```